core/json/src/main/scala/net/liftweb/json/JsonAST.scala

```diff
@@ -172,8 +277,11 @@ object JsonAST {
       rec(z, this)
     }
 
-    /** Return a combined value by folding over JSON by applying a function <code>f</code>
-     * for each field. The initial value is <code>z</code>.
+    /**
+     * Fold over a sries of `JField`s applying a function to each one.
+     *
+     * @param z The initial value for the fold.
+     * @param f The Function to apply, which takes an accumulator as its first param and the next field as its second.
```

Comment:
Is `Function` capitalized for a reason here, or is that a typo?

May also be worth saying “as its first parameter” rather than “as its first param”.